src/index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" media="screen" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <header class="header">
+        <div class="header-container">
+            <nav class="nav-item1"><a class="Mylogo" href="">My Logo</a></nav>
+            <nav class="nav-item2"></nav>
+            <nav class="nav-item3">
+                <img class="toggle" src="../assets/toggle.png" alt="toggle" onclick="openNav()">
+            </nav>
```

Comment:
- [ ] Great job so far :muscle: But as the previous reviewer mentioned, the mobile menu has a scroll effect which causes the close button to be out of the user's view when the menu is opened so kindly fix that so your design looks similar to that of the Figma design and has a better user interaction and experience :100: 
- [ ] I cloned your project and tested it out as a result of your earlier comment to the previous reviewer but I also had that effect. I have attached a video below for your reference :wink: 
- Also the video you attached in your comment redirected to an error page when I tried to view it :wink: 
[Screencast from 03-13-2023 10:33:53 PM.webm](https://user-images.githubusercontent.com/102699958/224848458-6dfabaa9-1391-4f0e-b5ca-64e2281d77ec.webm)
- [ ] Lastly, as the previous reviewer mentioned please ensure that your linters are correctly set up for Javascript, HTML, and CSS, kindly use these [configurations](https://github.com/microverseinc/linters-config/tree/master/html-css-js) :wink: Because of the conflicts it isn't possible to see the linter checks so I will suggest that you fix the merge conflicts :wink: